jest/preset.js

```diff
@@ -0,0 +1,10 @@
+const reactNativePreset = require('react-native/jest-preset');
```

Comment:
Is this being exposed as a jest preset of `@testing-library/react-native`? If so, it should be documented somewhere

Reply:
Yup, will do in a followup